modules/internal/ChapelGpuSupport.chpl

```diff
@@ -42,4 +45,10 @@ module ChapelGpuSupport {
           for gpu2 in here.gpus do
             if canAccessPeer(gpu1,gpu2) then setPeerAccess(gpu1,gpu2,true);
     }
+
+  private proc isEnvSet(name: string): bool {
```

Comment:
This seems sufficiently general that I'd be surprised if we didn't already have something in our standard modules that did it.

It looks like you could do this by looking at the return value of: `Sys.sys_getenv`

https://chapel-lang.org/docs/modules/standard/Sys.html?highlight=getenv#Sys.sys_getenv

Although the comment there tells you that, that's deprecated and to use: `OS.sys_getenv`.

Funny enough if I look at the doc for the `OS` module (https://chapel-lang.org/docs/modules/standard/OS.html) I don't see `sys_getenv` documented there. So maybe its no doc'd?

Anyway, you might want to see if you could use `OS.sys_getenv` instead and could you poke whoever is responsible for stabilizing the sys and/or OS module? Seems like we have an issue in our doc about it if nothing else.

Reply:
Good catch! For posterity: I brought this to the attention of the 2.0 subteam. I'll click resolve on this convo as there's not much I can do at the moment. But I am hoping that this will be fixed once we have a standard way of reading an environment variable.